corehq/apps/export/static/export/js/export_list_main.js

```diff
@@ -31,6 +33,14 @@ hqDefine("export/js/export_list_main", [
             }));
             $('#createExportOptionsModal').on('show.bs.modal', function () {
                 kissmetricsAnalytics.track.event("Clicked New Export");
+
+                const modelType = utils.capitalize(initialPageData.get('model_type', 'unknown'));
+                const metricsMessage = `${modelType} Export - Clicked Add Export Button`;
+                kissmetricsAnalytics.track.event(metricsMessage, {
```

Comment:
i think this is probably ok. as long as the data is there in some form, then it can be used on the Kissmetrics side. I don't really see an alternative